setup.py

```diff
@@ -50,7 +50,6 @@
     """,
     license = 'BSD',
 
-    install_requires = ['Trac >= 0.12dev'],
```

Comment:
You may be interested in: https://trac-hacks.org/wiki/DevGuide#AssertMinimumTracVersionRequirement